pkg/client/images.go

```diff
@@ -45,7 +45,7 @@ func (i *images) Get(id string) (*types.Image, error) {
 	return &image, nil
 }
 
-func (i *images) Build(name, sourceTar, base, lang, provider, args string, mounts []string, force, noCleanup bool) (*types.Image, error) {
+func (i *images) Build(name, sourceTar, base, lang, provider, args string, mounts []string, force, noCleanup bool, size string) (*types.Image, error) {
```

Comment:
Yes. But wouldn't it be great for user to be able to pick logical size on her own? Imagine VirtualBox where you specify maximum size for your .vdi HDD:
![capture](https://cloud.githubusercontent.com/assets/8102426/19600827/05c5bada-97a7-11e6-85ca-ab9d00058bcf.PNG)

I don't think this feature is special to this specific compiler. Any compiler could build image of arbitrary size, am I right? Of course, they don't do it yet, but they could.

Should you insist on removing this from the API, we could alternatively read this value from `manifest.yaml`. Like this:
```yaml
# manifest.yaml
image_size: 10GB
```
Would you prefer us to go this way?